src/routes/retro-certs.test.js

```diff
@@ -365,4 +365,66 @@ describe("Router: API tests", () => {
       upsertFormDataMock.mockRestore();
     }
   });
+
+  describe("retro-certs staff view login", () => {
+    const randomIpAddress = "70.191.9.39";
+    const eddIpAddress = "192.168.2.33";
+    const env = Object.assign({}, process.env);
+    env.ALLOWED_IP_RANGES = "192.168.2.1-192.168.2.100";
+    fflip.features.retroCerts.enabled = true;
```

Comment:
doesn't really matter since it's tested elsewhere, but since this feature flag gets set by init(env), here you can just set `env.ENABLE_RETRO_CERTS = "1"`

Reply:
Oh yeah, I meant to comment about that! I tried setting the status that way, but I couldn't get it to work. I don't really know how it's working in app.test.js. When I set the variable equal to "0" on line 46, the test still passed 😬 

I don't think we should spend too much time worrying about it, I created a ticket to remove the feature flag. Now that everything is enabled in prod, we don't need the feature flag anymore anyway.